docs/en/writing-running-appium/caps.md

```diff
@@ -137,6 +137,8 @@ Appium tries to guess the version of the WebView based on the version of the cor
 |`buildToolsVersion`| Specify the Android `build-tools` version to be something different than the default, which is to use the most recent version. It is helpful to use a non-default version if your environment uses alpha/beta build tools. | e.g. `'28.0.3'` |
 |`androidNaturalOrientation`| Allow for correct handling of orientation on landscape-oriented devices. Set to `true` to basically flip the meaning of `PORTRAIT` and `LANDSCAPE`. Defaults to `false` | `true`, `false` |
 | `enforceAppInstall` | By default application installation is skipped if newer or the same version of this app is already present on the device under test. Setting this option to `true` will enforce Appium to always install the current application build independently of the currently installed version of it. Defaults to `false`. | `true` , `false` |
+| `ignoreHiddenApiPolicyError` | Ignores `Security exception: Permission denial` alert to proceed the create session process since Appium 1.18.0+.. The error happens when Appium try to relax [hidden API policy](https://developer.android.com/distribute/best-practices/develop/restrictions-non-sdk-interfaces#how_can_i_enable_access_to_non-sdk_interfaces) for Appium feature. But some devices restrict the permission. Defaults to `false`. | `true`, `false` |
```

Comment:
double dot (..)

Reply:
... alert and allows to proceed to the session creation since Appium 1.18.0+

Reply:
...The error happens when Appium tries to relax...

Reply:
...for Appium feature. But some devices restrict the permission ->, although some devices with a customized firmware deny such requests